config.example.ini

```diff
@@ -0,0 +1,16 @@
+[DEFAULT]
+# your telegram bot api token
+BotToken=abcdef
+# webhook url, leave blank when you want to use polling instead
+WebhookUrl=
+# chat id for bot admin
+AdminId=1234
+# (optional) comma separated list of chat ids of developers
+# who get notified when an error occurs, default: <AdminId>
+DeveloperIds=1234,54321
```

Comment:
can we change this somehow so that the default is actually the default or that is line is by default commented out?